tm2/pkg/sdk/auth/handler.go

```diff
@@ -72,6 +79,22 @@ func (ah authHandler) queryAccount(ctx sdk.Context, req abci.RequestQuery) (res
 	return
 }
 
+// queryGasPrice fetch a gas price of the last block.
+func (ah authHandler) queryGasPrice(ctx sdk.Context, req abci.RequestQuery) (res abci.ResponseQuery) {
```

Comment:
Long-term, I think we should expose an API for fetching any kind of module param, but this is fine for now

Reply:
Sure, we will export it when the use case comes up.